lib/interpreters/split_clustering.rb

```diff
@@ -1008,12 +1005,18 @@ def self.duplicate_vehicle(vehicle, timewindow, schedule)
       available_days.collect{ |day|
         next unless (schedule[:start]..schedule[:end]).any?{ |day_index| day_index % 7 == day }
 
-        tw = timewindow ? Marshal.load(Marshal.dump(timewindow)) : Models::Timewindow.create({})
+        tw = Models::Timewindow.create(timewindow&.as_json&.symbolize_keys || {})
```

Comment:
`nil.as_json => nil` and keys are already symbol

```suggestion
        tw = Models::Timewindow.create(timewindow.as_json || {})
```